tests/core/math/test_transform_2d.h

```diff
@@ -45,6 +45,90 @@ Transform2D identity() {
 	return Transform2D();
 }
 
+TEST_CASE("[Transform2D] Default constructor") {
+	Transform2D default_constructor = Transform2D();
+	CHECK(default_constructor == Transform2D(Vector2(1, 0), Vector2(0, 1), Vector2(0, 0)));
+}
+
+TEST_CASE("[Transform2D] Copy constructor") {
+	Transform2D T = create_dummy_transform();
+	Transform2D copy_constructor = Transform2D(T);
+	CHECK(T == copy_constructor);
+}
+
+TEST_CASE("[Transform2D] Constructor from angle and position") {
+	const float ROTATATION = Math_PI / 4;
+	const Vector2 TRANSLATION = Vector2(20, -20);
+
+	Transform2D test = Transform2D(ROTATATION, TRANSLATION);
+	Transform2D expected = Transform2D().rotated(ROTATATION).translated(TRANSLATION);
+	CHECK(test == expected);
+}
+
+TEST_CASE("[transform2d] Constructor from angle, scale, skew and position") {
+	const float ROTATATION = Math_PI / 2;
```

Comment:
```suggestion
	const float ROTATION = Math_PI / 2;
```
and likewise where it's used.